### PR TITLE
PostgreSQL database for inntekt API - PART 2

### PR DESCRIPTION
**Pull request by geiralund**
- Use unleash to toggle on/off postgresql inntekt store
- Removed Environment class - use konfig lib (https://github.com/npryce/konfig) for application configuration
- Added README on howto start postgresql locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,79 @@
...
+Starte Postgres: 
+```
+
+docker-compose -f docker-compose.yml up
+
+```
+Etter at containerene er startet kan man starte applikasjonen ved å kjøre main metoden.
+
+
+Stoppe Kafka og Postgres:
```
Comment (gtcno) [inline README.md]: Nit: No Kafka

File: README.md [hunk 2]
```diff
@@ -1,2 +1,79 @@
...
+export VAULT_ADDR=https://vault.adeo.no USER=NAV_IDENT
+vault login -method=oidc
+
+
+```
+
+Preprod credentials:
+
+```
+vault read postgresql/preprod-fss/creds/dp-regel-api-preprod-admin
```
Comment (gtcno) [inline README.md]: Wrong path. Should be dp-inntekt-db-preprod?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,79 @@
...
+
+```
+
+Prod credentials:
+
+```
+vault read postgresql/prod-fss/creds/dp-regel-api-admin
```
Comment (gtcno) [inline README.md]: Wrong path. Should be dp-inntekt-db?

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+version: '3'
+services:
+  postgres:
+    image: postgres:11
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=dp-regel-api
```
Comment (gtcno) [inline docker-compose.yml]: Wrong POSTGRES_DB name. Should match db name in LOCAL configuration  prop map(dp-inntekt-db)

File: src/main/kotlin/no/nav/dagpenger/inntekt/Configuration.kt [hunk 5]
```diff
@@ -27,7 +31,11 @@ private val devProperties = ConfigurationMap(
                 "database.name" to "dp-inntekt-db-preprod",
                 "vault.mountpath" to "postgresql/preprod-fss/",
                 "application.profile" to "DEV",
-                "application.httpPort" to "8099"
+                "application.httpPort" to "8099",
+                "hentinntektliste.url" to "https://localhost/inntektskomponenten-ws/rs/api/v1/hentinntektliste",
```
Comment (gtcno) [inline src/main/kotlin/no/nav/dagpenger/inntekt/Configuration.kt]: You might consider removing these "default" config values  from DEV. The application will then fail fast with a nice error message if these config values are missing from the environment.

File: src/main/kotlin/no/nav/dagpenger/inntekt/Configuration.kt [hunk 6]
```diff
@@ -38,7 +46,11 @@ private val prodProperties = ConfigurationMap(
                 "database.name" to "dp-inntekt-db",
                 "vault.mountpath" to "postgresql/prod-fss/",
                 "application.profile" to "PROD",
-                "application.httpPort" to "8099"
+                "application.httpPort" to "8099",
+                "hentinntektliste.url" to "https://localhost/inntektskomponenten-ws/rs/api/v1/hentinntektliste",
```
Comment (gtcno) [inline src/main/kotlin/no/nav/dagpenger/inntekt/Configuration.kt]: You might consider removing these "default" config values  from PROD. The application will then fail fast with a nice error message if these config values are missing from the environment.

File: README.md (hunk 1 above)
Comment (geiralund) [reply README.md]: oh, will fix

File: README.md (hunk 2 above)
Comment (geiralund) [reply README.md]: doh..

File: src/main/kotlin/no/nav/dagpenger/inntekt/Configuration.kt (hunk 5 above)
Comment (geiralund) [reply src/main/kotlin/no/nav/dagpenger/inntekt/Configuration.kt]: good point..

File: README.md (hunk 3 above)
Comment (geiralund) [reply README.md]: yep, fixed..

File: docker-compose.yml (hunk 4 above)
Comment (geiralund) [reply docker-compose.yml]: fixed!

File: src/main/kotlin/no/nav/dagpenger/inntekt/Configuration.kt (hunk 5 above)
Comment (geiralund) [reply src/main/kotlin/no/nav/dagpenger/inntekt/Configuration.kt]: fixed

File: src/main/kotlin/no/nav/dagpenger/inntekt/Configuration.kt (hunk 6 above)
Comment (geiralund) [reply src/main/kotlin/no/nav/dagpenger/inntekt/Configuration.kt]: fixed!

**Pull request merged**